gluoncv/model_zoo/rcnn/faster_rcnn/doublehead_rcnn.py

```diff
@@ -0,0 +1,642 @@
+"""
+Double Head Faster RCNN Model.
```

Comment:
Can you add more information, like the author information and license information? Thank you.

Reply:
@bryanyzhu thank you.  i have modified the related information. 
i have another question with workflow error in https://github.com/dmlc/gluon-cv/pull/1727
thanks.

Reply:
@bryanyzhu how to fix this error. thanks.

======================================================================
ERROR: tests.unittests.test_data_datasets.test_lst_detection
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/miniconda/envs/gluon_cv_py3_test/lib/python3.6/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/runner/work/gluon-cv/gluon-cv/tests/unittests/test_data_datasets.py", line 111, in test_lst_detection
    path='dog.jpg')
  File "/Users/runner/work/gluon-cv/gluon-cv/gluoncv/utils/download.py", line 69, in download
    raise RuntimeError("Failed downloading url %s"%url)
RuntimeError: Failed downloading url https://github.com/dmlc/web-data/blob/master/gluoncv/datasets/dog.jpg?raw=true
-------------------- >> begin captured stdout << ---------------------
Downloading dog.jpg from https://github.com/dmlc/web-data/blob/master/gluoncv/datasets/dog.jpg?raw=true...

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): github.com:443
urllib3.connectionpool: DEBUG: https://github.com:443 "GET /dmlc/web-data/blob/master/gluoncv/datasets/dog.jpg?raw=true HTTP/1.1" 429 1474
--------------------- >> end captured logging << ---------------------